h2o-r/tests/testdir_algos/automl/runit_automl_args.R

```diff
@@ -1,5 +1,6 @@
 setwd(normalizePath(dirname(R.utils::commandArgs(asValues=TRUE)$"f")))
 source("../../../scripts/h2o-r-test-setup.R")
+source("../../runitUtils/matchers.R")
```

Comment:
Remove.

Reply:
Yes, not needed.

Reply:
I would ask you to give me objective reason to not to use new methods. 'To simplify and for consistency' is somewhat subjective. Maybe this comes from the community that is used to **testthat** api? Is this the real reason? Just share with me your conventions :)